Harivigneshkumar/Contact Form 1/contactform.html

```diff
@@ -0,0 +1,112 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Contact Form</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
+    </head>
+<body>
+    
+    <div class="heading">
+        <h1>Contact Form</h1>
+    </div>
+        <form action="index.html" method="get" target="_blank">
+    <div class="form1">
+        <label>First Name  :</label><input type="text" class="value" id="fname" placeholder="Enter your First Name"><br><br>
+        <label>Last Name :</label><input type="text" class="value" id="lname" placeholder="Enter your Last Name"><br><br>
+        <label>Mobile Number :</label><input type="text" class="value" id="mobilenumber" placeholder="Enter your Mobile Number"><br><br>
+        <label>Date of Birth :</label><input type="date" class="value" id="dob" placeholder="Enter your date of birth"><br><br>
+        <label>Gender :</label><select id="Gender" class="value">
+            <option value="male">male</option>
+            <option value="female">female </option></select><br><br>
+        <label>Email :</label><input type="email" class="value" id="email" placeholder="Enter Your Email Id" required><br><br> 
+        <label>Website :</label><input type="url" id="website" class="value" placeholder="Enter Your Website Name"><br><br>
+        <label>Groups :</label><select id="Groups" class="value">
+            <option value="Important">Important</option>
+            <option value="Emergency">Emergency </option>
+            <option value="Not Assigned">Not Assigned </option></select><br><br>
+            <label>Messager Accounts :</label>
+            <select id="messager" class="value">
+                <option value="Hangouts">Hangouts</option>
+                <option value="Skype">Skype </option>
+                <option value="Windows Live">Windows Live </option>
+                <option value="Yahoo">Yahoo </option>
+                <option value="QQ">QQ </option>
+            </select><br><br>
+        <label>State :</label><select id="state" class="value">
+        <option value="TamilNadu">TamilNadu </option>
+        <option value="AndhraPradesh">AndhraPradesh </option>
+        <option value="Kerala">Kerala </option>
+        <option value="Karnataka">Karnataka </option>
+        </select><br></br>
+        <label>Address :</label>
+        <input type="text" class="value" id="address" placeholder="Enter your Address"><br><br>
+        <label>Work Info :</label>
+        <input type="text" class="value" id="work"placeholder="Enter your Job Title"><br>
+        <button type="submit" value="click" onclick="myfun()" class ="accept">Save</button>
+        <button type="reset" value="click"  class ="regret">Cancel</button>
+        <br><br><br></div></form>
+        <script>
+            function myfun(){
```

Comment:
Move this Script code to separate JS file and provide link in the HTML page.
Indent the code properly in JS file.